### DISCUSSION TITLE
Merging multiple atac samples before pre-processing

### DISCUSSION THREAD

**Discussion by tangybat** [welch-lab/MultiVelo#21] [Q&A]
Hi,

I have atac data from multiple samples. Before preprocessing, when I try to merge the filtered_feature_bc_matrices from my samples using .concatenate, n_vars becomes empty (I suspect due to no matching gene IDs; the gene ID names for peaks are all in the format where they end in some range of specific numbers as in the screenshot below). 
`adata_atac = atac_10.concatenate(atac_19, atac_2, atac_27, atac_31, atac_35, atac_38, atac_4, atac_43, atac_46, atac_47, atac_5, atac_7)`
<img width="391" alt="image" src="https://user-images.githubusercontent.com/86893865/225199624-2bd39cea-0cf2-4ce8-9484-19a31651d3e3.png">
Do you know how to merge these atac files without losing data?

On a separate note, after filtering each of my rna samples (spliced/unspliced loom files generated from kallisto pipeline) based on the cell IDs/barcodes extracted from my combined seurat object, I was only left with ~900 cells out of ~62000 potential cells from my combined seurat object after concatenating the rna samples together. I was then only left with 167 cells after filtering via 
`sc.pp.filter_cells(adata_rna, min_counts=1000)`. Is this unusual? Thank you!

**Comment by danielee0707** [comment]
Hi, Signac package in R has some functions to merge ATAC-seq objects [.](https://stuartlab.org/signac/articles/merging.html). You'll need to first find a common peak set. You can then read the combined count matrix into python.
And for the RNA-seq data, it doesn't sound usual if most cells were filtered out. There are probably some issues with the code or the datasets themselves. Have you run any quality control preprocessing steps using seurat, scanpy, or scvelo to see what the samples were like? And which field in the seurat object was your filtering based on?
